contracts/SoulboundIdentity.sol

```diff
@@ -30,20 +24,23 @@ contract SoulboundIdentity is
     /// @param symbol Symbol of the token
     /// @param baseTokenURI Base URI of the token
     /// @param paymentParams Payment gateway params
+    /// @param maxSBTToMint Maximum number of SBT that can be minted
     constructor(
         address admin,
         string memory name,
         string memory symbol,
         string memory baseTokenURI,
-        PaymentParams memory paymentParams
+        PaymentParams memory paymentParams,
+        uint256 maxSBTToMint
```

Comment:
can be removed and hardcoded to 1, there is no known case where something > 1 was used here